ui/src/reducers/index.js

```diff
@@ -27,9 +27,9 @@ const initialState = {
     { title: 'App Version', transforms: [sortable] },
     { title: 'Current Installs', transforms: [sortable] },
     { title: 'Launches', transforms: [sortable] },
-    { title: 'Last Launched', transforms: [sortable] },
+    { title: 'Last Launched', transforms: [sortable, cellWidth(25)] },
```

Comment:
Should we add `cellWidth` for all the columns to ensure the proportions remain consistent regardless of screen width?

Reply:
@damienomurchu I did play with this. The columns default to the size of the column header. As the screen gets smaller on the x axis the columns with cellWidth(25) shrink and wrap text in the cells until the size of the header. Once all columns are as small as to still be able to show the full header the table doesn't shrink any further and the user can then scroll. As far as I can see the cellWidth is maintained only when the screen size will allow. Otherwise the table is responsive to the header text.

Reply:
Understand. Explicit is better than implicit I think, but happy for this to stay the same.